crates/router/src/core/payments/transformers.rs

```diff
@@ -574,6 +576,13 @@ impl<F: Clone> TryFrom<PaymentAdditionalData<'_, F>> for types::PaymentsAuthoriz
             router_return_url,
             webhook_url,
             complete_authorize_url,
+            country_code: billing_address.country,
+            address_line1: billing_address.line1,
+            address_line2: billing_address.line2,
+            address_line3: billing_address.line3,
+            first_name: billing_address.first_name,
+            last_name: billing_address.last_name,
+            postal_code: billing_address.zip,
```

Comment:
All these fields are present in `RouterData`, these seems to be repeated here
https://github.com/juspay/hyperswitch/blob/a6cc8e08c560bec2e59c7c7290a6382d1611e8df/crates/router/src/types.rs#L180